include/tvm/relay/pass.h

```diff
@@ -403,6 +412,97 @@ TVM_DLL Expr ToGraphNormalForm(const Expr& e);
  */
 TVM_DLL Expr PartialEval(const Expr& e);
 
+/*!
+ * \brief Bind the free variables to a Relay expression.
+ *
+ * \param expr The expression.
+ * \param bind_map The variable to expression map that will be used to help the
+ *        binding.
+ *
+ * \return The updated expression.
+ */
+TVM_DLL Expr Bind(const Expr& expr, const tvm::Map<Var, Expr>& bind_map);
+
+/*!
+ * \brief Simplify certain operators during inference. For example, batch norm
+ * will be unpacked into a number of simplified operators.
+ *
+ * \param expr The expression.
+ *
+ * \return The updated expression.
+ */
+TVM_DLL Expr SimplifyInference(const Expr& expr);
+
+/*!
+ * \brief Search and eliminate common subexpression. For example, if there are
+ * two expressions evaluated to an identical value, a single variable is created
+ * and these two expressions are replaced by this variable.
+ *
+ * \param expr The expression.
+ * \param fskip The callback argument that allows to skip certain expressions.
+ *
+ * \return The updated expression.
+ */
+TVM_DLL Expr EliminateCommonSubexpr(const Expr& expr, PackedFunc fskip);
+
+/*!
+ * \brief Combine parallel 2d convolutions into a single convolution if the
+ * number of branches of this conv2d operator is not less than
+ * `min_num_branch`.
+ *
+ * \param expr The expression.
+ * \param min_num_branches The minimun number of branches.
+ *
+ * \return The updated expression.
+ */
+TVM_DLL Expr CombineParallelConv2D(const Expr& expr, uint64_t min_num_branches);
+
+namespace fold_scale_axis {
+
+/*!
+ * \brief Backward fold axis scaling into weights of conv/dense operators.
+ *
+ * \param expr The input expression.
+ *
+ * \return The updated expression. 
+ */
+TVM_DLL Expr BackwardFoldScaleAxis(const Expr& expr);
+
+/*!
+ * \brief Forward fold axis scaling into weights of conv/dense operators.
+ *
+ * \param expr The input expression.
+ *
+ * \return The updated expression. 
+ */
+TVM_DLL Expr ForwardFoldScaleAxis(const Expr& expr);
```

Comment:
We should start deprecating these functions, which means we should remove them  pass.h and move to only rely on transform.h